lib/utils.js

```diff
@@ -9,13 +10,14 @@ const c = require('chalk');
 const ssLog = require('single-line-log');
 const pad = require('pad');
 const {
-  keyBy, prop, merge, isEmpty, keys, union, find, propEq, propOr
+  keyBy, prop, merge, isEmpty, keys, union, find, propEq, propOr, mergeRight, has
 } = require('omnibelt');
 const { rollbarLog } = require('./rollbar');
 const CONFIG_DIR = '.losant';
 const configName = (file) => {
   return file || '.application.yml';
 };
+const DEFAULT_URL = process.env.DEFAULT_API_URL || 'https://api.losant.com';
```

Comment:
put this at the very top of the file.

Reply:
actually no put this in the entry file, of losant.js in the bin, right below process.env.NODE_ENV.